pkg/cmd/cli/cmd/create.go

```diff
@@ -99,9 +99,11 @@ func (o *CmdOptions) RunCreate(out io.Writer, cmd *cobra.Command, args []string)
 		return err
 	}
 	if result.Ephemeral == "<shared>" {
-		fmt.Fprintf(out, "shared cluster \"%s\" created \n", o.Name)
+		fmt.Fprintf(out, "shared cluster %s created \n", o.Name)
+		fmt.Fprintf(out,"spark master: %v", result.MasterURL+"\n")
 	} else {
-		fmt.Fprintf(out, "ephemeral cluster \"%s\" created \n", o.Name)
+		fmt.Fprintf(out, "ephemeral cluster %s created \n", o.Name)
+		fmt.Fprintf(out,"spark master: \"%s\"", result.MasterURL+"\n")
```

Comment:
I'm late to the game, but is there a reason for using %v above and %s here?

Reply:
originally i was solving a problem which i thought was the type but in fact i was missing a comma i can change this back to %s